seatunnel-engine/seatunnel-engine-ui/cypress/e2e/example.cy.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// https://on.cypress.io/api
+
+describe('My First Test', () => {
```

Comment:
Hi, how to trigger test case in ci. I'm not familiar with the web.

Reply:
Use "npm run test" to run the test cases. I did not use Cypress and have just removed it. Now, Vitest is being used as the testing scaffold.